easybuild/easyconfigs/b/binutils/binutils-2.25.eb

```diff
@@ -19,8 +19,10 @@ builddependencies = [
 ]
 
 # statically link with zlib, to avoid runtime dependency on zlib
-preconfigopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
-prebuildopts = 'LIBS="$EBROOTZLIB/lib/libz.a"'
+# further, add the system library path in the rpath: this should 'harden' the
+# resulting binutils to bootstrap GCC (no trouble when other libstdc++ is build etc)
+preconfigopts = 'LIBS="$EBROOTZLIB/lib/libz.a -Wl,-rpath=/lib64 -Wl,-rpath=/usr/lib64 -Wl,-rpath=/lib -Wl,-rpath=/usr/lib"'
```

Comment:
@wpoely86: let's leave it as is for now, since this is tested thoroughly, and should be good to merge

We can clean it up later.

I'd love to get feedback from @geimer on this though, before merging it...